### ISSUE TITLE
Error in installing pod with cocoapods 1.15.1 - FBAudienceNetwork 6.14.0

### ISSUE THREAD

**Issue by AlexandrMurashko** [CocoaPods/CocoaPods#12268]
* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

## What did you do?

Run `pod FBAudienceNetwork`

## What did you expect to happen?

Install all pod dependencies correctly.

## What happened instead?

File not found 'Info.plist' upon install pod
The plist file at path `Pods/FBAudienceNetwork/Static/FBAudienceNetwork.xcframework/Info.plist` doesn't exist.

### Stack

```
   CocoaPods : 1.15.1
        Ruby : ruby 3.3.0 (2023-12-25 revision 5124f9ac75) [arm64-darwin23]
    RubyGems : 3.5.4
        Host : macOS 14.3 (23D56)
       Xcode : 15.2 (15C500b)
         Git : git version 2.39.3 (Apple Git-145)
Ruby lib dir : /opt/homebrew/Cellar/ruby/3.3.0/lib
Repositories : trunk - CDN - https://cdn.cocoapods.org/
```

### Installation Source

```
Executable Path: /opt/homebrew/Cellar/cocoapods/1.15.1/libexec/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.5
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.1
cocoapods-trunk       : 1.6.0
cocoapods-try         : 1.2.0
```

**Comment by AlexandrMurashko** [comment]
Good work with cocoapods 1.14.3

**Comment by paulb777** [comment]
Fixed in 1.15.2
